### DISCUSSION TITLE
Weekly Governance Updates (11 November 2022)

### DISCUSSION THREAD

**Discussion by luckyparadise** [filecoin-project/FIPs#522] [General]
This weekly report provides a high-level overview of what's happening with Filecoin governance.

Anyone is welcome to participate in governance. To learn more about any of the below materials, simply follow the links and feel free to weigh in.

If you have questions or ideas related to FIPs or Filecoin governance, please start a new discussion in the FIPs Forum or post directly in [#fil-fips](https://github.com/filecoin-project/FIPs/discussions/filecoinproject.slack.com).

---

**Accepted FIPs** - *FIPs awaiting network implementation*

- FIP0027 - Change type of DealProposal Label field from a (Golang) String to a Union [(FIP)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0027.md)[(Discussion)](https://github.com/filecoin-project/FIPs/issues/187)
- FIP0029 - Beneficiary address for storage providers [(FIP)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0029.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/253)
- FIP0034 - Set pre-commit deposit independent of sector content [(FIP)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0034.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/290)
- FIP0041 - Forward Compatibility for PreCommit and ReplicaUpdate [(FIP)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0041.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/380)
- FIP0044 - Standard Authentication Method for Actors [(FIP)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0044.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/413)
- FIP0045 - De-couple verified registry from markets [(FIP)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0045.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/313)
- FIP0047 - Proof Expiration & PoRep Security Policy [(FIP)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0047.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/415)

**Last Call** - *Final opportunity for community feedback!* 

NA

**Current FIP Drafts** - *Now accepting community feedback; merged into repo*

- FIP0037 - Gas Model Adjustment for User Programmability [(FIP Draft)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0037.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/316)
- FIP0039 - Filecoin Message Replay Protection [(FIP Draft)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0039.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/301)
-  FIP0048 - f4 Address Class [(FIP Draft)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0048.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/473)

**Open Pull Requests** - *upcoming FIP drafts awaiting technical review; now accepting community feedback*

- FIP0043 - Introduce Domain Separation - [(FIP)](https://github.com/filecoin-project/FIPs/discussions/387)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/387%20status:%20Draft)
- FIP0049 - Actor Events - [(FIP)](https://github.com/filecoin-project/FIPs/pull/483)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/484)

**🔥 HOT 🔥 Discussion Topics**

- @Kubuxu started a conversation on Verifiable Off-Chain Data aggregation. You can read the discussion in detail [here](https://github.com/filecoin-project/FIPs/discussions/512) 
- @ZenGround0 put together thoughts on configurable sector deadline/partition assignment. This would help SPs in a number of ways. You can read the discussion in detail [here](https://github.com/filecoin-project/FIPs/discussions/505)

**Other Governance Updates**

- Plans are rapidly unfolding for nv17 - to stay in the loop, be sure to check out the latest _Shark_  Network nv17 timelines [here](https://github.com/filecoin-project/community/discussions/74?sort=top#discussioncomment-3825422)!
- Dont forget to share feedback in our governance survey for the Filecoin community. The core aim is to  provide space for community members to provide feedback on their experience and priorities for the Filecoin Foundation governance team. The survey can be found [here](https://docs.google.com/forms/d/1Cn4XTwI2HSbAzKRuZoYahiGqwi6c1Cla9CGf8OlydeI/edit?ts=6345ca05). We look forward to your responses.

**Comment by anorth** [comment]
I believe FIP-0027 went in [nv16](https://github.com/filecoin-project/tpm/blob/master/Network%20Upgrades/v16.md).
